kdb/key.go

```diff
@@ -348,6 +365,8 @@ func (k *ckey) Namespace() string {
 	return name[:index]
 }
 
+// TODO API REVIEW: Do we really need this?
```

Comment:
We could un-export it?

Reply:
Which function called NameWithoutNamespace is exported by Elektra?

Reply:
i meant unexport it from the go package - it is used internally in the CommonKeyName function

Reply:
Yes, maybe better unless you see a good reason why do have it exported (e.g. you need it in your app).